app/src/main/java/org/schabi/newpipe/fragments/list/comments/CommentsFragment.java

```diff
@@ -27,7 +31,7 @@
 public class CommentsFragment extends BaseListInfoFragment<CommentsInfoItem, CommentsInfo> {
```

Comment:
I've already migrated this fragment to Compose in [a separate PR](https://github.com/TeamNewPipe/NewPipe/pull/11060), so I don't think this particular change will be needed.

Reply:
Good to know. This can wait for your PR.

Reply:
This has been adjusted in the rebase